### PR TITLE
Added Problem 482

### LINKED ISSUE #26: add problems 479-500
* [ ] 479-489
* [ ] 490-500

### PR DESCRIPTION
**Pull request by georgenetu**
I like the changes from this "updated" PocketEuler version, so I'm doing my best to close #26 starting with today.

### CODE REVIEW

**Comment by georgenetu** [comment]
What should I do in this case?

**Comment by jayvdb** [comment]
Hi @georgenetu , the failure isnt caused by you.
It is caused by a new release of the checker tools.  I will investigate it now, and submit a patch to fix it.

**Comment by jayvdb** [comment]
Sorry for the delay; #58 gets the build working again.

File: README.md [hunk 1]
```diff
@@ -12395,3 +12395,24 @@ Find E(14) rounded to 8 decimal places.
...
+Problem 482
+===========
+
+ABC is an integer sided triangle with incenter I and perimeter p.
+
+The segments IA, IB and IC have integral length as well.
+
+Let L = p + |IA| + |IB| + |IC|.
+
+Let S(P) = ∑L for all such triangles where p ≤ P. For example, S(103) = 3619.
```
Comment (jayvdb) [inline README.md]: Should be S(10^3)

File: README.md [hunk 2]
```diff
@@ -12395,3 +12395,24 @@ Find E(14) rounded to 8 decimal places.
...
+
+Find S(107).
```
Comment (jayvdb) [inline README.md]: S(10^7)

**Comment by georgenetu** [comment]
Thanks for the quick fix @jayvdb.

**Pull request merged**